views/js/component/diagnostic/tpl/component.tpl

```diff
@@ -0,0 +1,3 @@
+<div class="diagnostics-content-area">
+    <h1>Diagnostic Tool</h1>
```

Comment:
This should be translated: `{{__ 'Diagnostic Tool'}}`. This is why the translation is lost in the `*.po` files.
Don't forget to reapply an update of translations after.